pyspecProcScripts/load_data.py

```diff
@@ -346,6 +330,11 @@ def proc_Hahn_echoph(s, fl=None):
     return s
 
 
+def proc_spincore_FID_nutation_v1(s, fl=None):
+    s = proc_spincore_generalproc_v1(s, fl=fl)
+    return s
+
```

Comment:
while this may be the same as other postprocessing I figure it's nice to have the label match the experiment for clarity